build/build.sh

```diff
@@ -9,6 +9,15 @@ CMAKE=cmake
 CMAKE_MAJOR_VERSION=`cmake --version | head -n 1 | awk '{print $3}' |awk -F. '{print $1}'`
 CPU=`uname -m`
 
+#If git modules config file exist then try to clone them
+#Temporary fix for build pipeline to work
```

Comment:
```suggestion
# If git modules config file exist then try to clone them
# Temporary fix for build pipeline to work
```

I would like to understand the temporariness of this "fix".

Reply:
Hi @stsoe thank you for the review. Actually, I am not 100% sure if this is the right place to add these git commands, so added that comment as "temporary". Since, without the "git submodule" commands, build will will failed anyway.

Please let me know if this is the right place for these commands, then I will remove "Temporary" keyword line from build.sh script. Thank you.